components/Accordian/index.jsx

```diff
@@ -0,0 +1,203 @@
+      />
+      <div className="h-20 w-full pl-5 pt-2 flex items-start flex-col">
+        <h1 className="text-lg font-bold text-[#7A7A7A]">{jobTitle}</h1>
+        <div className="flex space-x-2 mt-2">
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Location Point.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">
+              {location}
+            </h1>
+          </div>
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Travel bag.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">{exp}</h1>
+          </div>
+        </div>
+      </div>
+
+      <div className="absolute top-7 right-5 text-[#7A7A7A] transition-transform duration-500 rotate-0 peer-checked:rotate-180">
+        {/* Arrow */}
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          className="h-6 w-6"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+          strokeWidth="2"
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            d="M19 9l-7 7-7-7"
+          />
+        </svg>
+      </div>
+      {/* Contents */}
+      <div className="max-h-0 peer-checked:max-h-fit  bg-white overflow-hidden transition-all duration-500">
+        <div className="space-y-5">
+          <div className="p-4">
+            <div className="space-y-3 ">
+              <div>
+                <h1 className="font-bold text-[#7A7A7A]">
+                  Role:{'  '}
+                  <span className="font-normal text-[#7A7A7A]">{jobTitle}</span>
+                </h1>
+              </div>
+              <div>
+                <h1 className="font-bold text-[#7A7A7A]">
+                  Location:
+                  <span className="font-normal text-[#7A7A7A]">{location}</span>
+                </h1>
+              </div>
+              <div>
+                <h1 className="font-bold text-[#7A7A7A]">
+                  Experience:
+                  <span className="font-normal text-[#7A7A7A]"> {exp}</span>
+                </h1>
+              </div>
+              <div>
+                <h1 className="font-bold text-[#7A7A7A]">
+                  Key Skills:
+                  <span className="font-normal text-[#7A7A7A]">
+                    Ruby on Rails, Mysql/NoSql
+                  </span>
+                </h1>
+              </div>
+            </div>
+            <div>
+              <h1 className="font-bold text-[#7A7A7A] mt-5">
+                What will your role look like
```

Comment:
Shift the h1 and li points to a JSON object with key as title and string array as value containing list items to render